torch/nested/__init__.py

```diff
@@ -397,7 +398,7 @@ def nested_tensor_from_jagged(
     if jagged_dim is None:
         jagged_dim = 1
 
-    from torch.nested._internal.nested_tensor import nested_view_from_values_offsets_lengths
-
-    return nested_view_from_values_offsets_lengths(
-        values, offsets, lengths, ragged_idx=jagged_dim, min_seqlen=min_seqlen, max_seqlen=max_seqlen)
+    # NB: the full path -must- be used here instead of importing so that @fx.wrap works
+    return torch.nested._internal.nested_tensor.nested_view_from_values_offsets_lengths(
```

Comment:
I think we need to also `import torch.nested._internal.nested_tensor`

Reply:
okay, apparently we can't do this, repro:

```python
import torch
import torch.nested._internal.nested_tensor

asdf = "asdf"

def fn(x, y):
    return torch.nested.nested_tensor_from_jagged(x, y)

x = torch.randn(10, 16)
y = torch.tensor([0, 1, 3, 6, 10])

fn_fx = torch.fx.symbolic_trace(fn)
torch.fx.symbolic_trace(fn_fx)
```

Reply:
i think this is why - fns from torch/ have some special handling here:

https://www.internalfb.com/code/fbsource/[4060bddc90fe1f9357b0a73656e0879678a364d1]/fbcode/caffe2/torch/fx/graph.py?lines=398-400